dbt_copilot_helper/commands/config.py

```diff
@@ -13,6 +13,10 @@
 no = "\033[91m✖\033[0m"
 maybe = "\033[93m?\033[0m"
 
+# Destination ARNs for log groups in the central logging account (for sending logs to Sentinal).
+PROD_LOG_GROUP_ARN = "cwl_log_destination_prod_to_be_updated_once_created"
+DEV_LOG_GROUP_ARN = "arn:aws:logs:eu-west-2:812359060647:destination:cwl_log_destination"
```

Comment:
Not sure what to do about account IDs, I have notices account IDs in other files eg, appconfig-ipfilter.

I thought it was general config, I could move the constants to copilot.py.  It would be nice to have a central place for this, but not sure what that should be

Reply:
hmm, maybe store the account IDs in github secrets